.github/workflows/tests.yml

```
@@ -85,7 +85,7 @@ jobs:
         mariadb_port: ${{ job.services.mariadb.ports[3306] }}
 
     - name: Upload coverage report
-      if: ${{ matrix.php-versions == '7.4' && matrix.db-types == 'mysql' }}
+      if: ${{ matrix.php-versions == '7.4' && matrix.db-types == 'mysql' && github.repository == 'mautic/mautic' }}
```

```suggestion
      if: ${{ matrix.php-versions == '7.4' && matrix.db-types == 'mysql' }}
```

This may not be desirable as folks may want to run GitHub Actions on their local fork to run unit tests and we need to run those tests in the security repo. The others deffo are OK but I think this one should be able to run in any repo.

---

@RCheesley this is the `Upload coverage report` step specifically, the unit tests will stil run, or is the upload of that report needed?

---

Ahhhh, sorry my bad, I didn't expand up the code snippet! In that case, yes all is good!